### DISCUSSION TITLE
Updating only some selcted particles in HPMC-NVT

### DISCUSSION THREAD

**Discussion by KaustavChakraborty** [glotzerlab/hoomd-blue#1751] [General]
Hello HOOMD users,
I want to read a configuration of N hard spheres inside a periodic box from a gsd file and want to perform NVT simulation in a way that selective N’ spheres will not move but participate in overlap checking while the rest of the spheres will move. Can anyone help me out to execute this in HOOMD?

**Comment by tcmoore3** [comment]
You need to make the mobile and immobile particles different types and set the move size of the immobile type to zero. See [this how-to guide](https://hoomd-blue.readthedocs.io/en/latest/howto/prevent-particles-from-moving.html) for an example.

**Reply by joaander** [reply]
Also see this tutorial: https://hoomd-blue.readthedocs.io/en/latest/tutorial/08-Placing-Barriers-in-the-Simulation-Box/00-index.html
